### PR TITLE
pacparser: fix compilation with clang

### PR DESCRIPTION
**Pull request by vvolkl**

### CODE REVIEW

File: var/spack/repos/builtin/packages/pacparser/package.py [hunk 1]
```diff
@@ -14,16 +14,19 @@ class Pacparser(MakefilePackage):
...
+
     depends_on('python', when='+python')
     depends_on('py-setuptools', when='+python', type=('build', 'run'))
 
     variant('python', default=False,
             description='Build and install python bindings')
 
     def build(self, spec, prefix):
-        make('-C', 'src')
+        make('CC=cc', 'CCC=c++', '-C', 'src')
```
Comment (iarspider) [inline var/spack/repos/builtin/packages/pacparser/package.py]: I think this is the Spack way of explicitly setting compiler. @tldahlgren correct me if I'm wroing.

```suggestion
        make('CC="%s"' % self.compiler.cc, 'CXX="%s"' % self.compiler.cxx, '-C', 'src')
```

File: var/spack/repos/builtin/packages/pacparser/package.py [hunk 2]
```diff
@@ -14,16 +14,19 @@ class Pacparser(MakefilePackage):
...
         if '+python' in spec:
-            make('-C', 'src', 'pymod')
+            make('CC=cc', 'CCC=c++', '-C', 'src', 'pymod')
```
Comment (iarspider) [inline var/spack/repos/builtin/packages/pacparser/package.py]:
```suggestion
            make('CC="%s"' % self.compiler.cc, 'CXX="%s"' % self.compiler.cxx, '-C', 'src', 'pymod')
```

File: var/spack/repos/builtin/packages/pacparser/package.py [hunk 3]
```diff
@@ -14,16 +14,19 @@ class Pacparser(MakefilePackage):
 
     version('1.3.7', sha256='eb48ec2fc202d12a4b882133048c7590329849f32c2285bc4dbe418f29aad249')
 
+    # possible race condition in makefile
+    parallel = False
```
Comment (iarspider) [inline var/spack/repos/builtin/packages/pacparser/package.py]: I did not see any issues building pacparser in parallel, but I didn't yet test it with clang.

```suggestion
    @property
    def parallel(self):
        return self.spec.satisfies('@1.3.7')
```

File: var/spack/repos/builtin/packages/pacparser/package.py [hunk 4]
```diff
@@ -14,16 +14,19 @@ class Pacparser(MakefilePackage):
 
     version('1.3.7', sha256='eb48ec2fc202d12a4b882133048c7590329849f32c2285bc4dbe418f29aad249')
```
Comment (iarspider) [inline var/spack/repos/builtin/packages/pacparser/package.py]: Please update your fork from upstream.

**Review by iarspider — CHANGES_REQUESTED** [review]

**Review by iarspider — DISMISSED** [review]
LGTM. Did not test with clang myself.

**Comment by iarspider** [comment]
@alalazo  @tldahlgren I don't have write access, can't merge it myself.

**Comment by iarspider** [comment]
@vvolkl 
> var/spack/repos/builtin/packages/pacparser/package.py:34: [E501] line too long (100 > 88 characters)

**Review by iarspider — APPROVED** [review]

**Review by tldahlgren — APPROVED** [review]

**Pull request merged**